todo-backend/TodoBackend/binaris.yml

```diff
@@ -0,0 +1,8 @@
+functions:
+  public_TodoBackend:
+    file: function.js
+    entrypoint: handler
+    runtime: node8
+    env:
+      BINARIS_API_KEY:
+      BINARIS_ACCOUNT_NUMBER:
```

Comment:
you should be able to remove this one. the environment during runtime will have BINARIS_ACCOUNT_ID working